resources/public/api/conf/5.0/schemas/foreign_property_cumulative_summary_create_and_amend/def1/request.json

```diff
@@ -1,17 +1,17 @@
 {
   "$schema": "http://json-schema.org/draft-04/schema#",
-  "title": "For TY 2025-26",
+  "title": "For TY 2025-26 and after",
   "type": "object",
   "properties": {
     "fromDate": {
       "type": "string",
-      "description": "The first day that the income and expenses period summary covers. Format: YYYY-MM-DD",
+      "description": "The first day that the income and expenses period summary covers. Must conform to the format: YYYY-MM-DD.\n\nFor users with ITSA status of 'annual' or a latent income source, these dates are not required. Endpoint will default the start and end dates to the tax year start and end dates. Default value: 06/04/TY -> 05/05/TY+1",
```

Comment:
This should probably be 06/04/TY -> 05/04/TY+1, if so the spec is also incorrect